core/src/main/java/org/elasticsearch/action/admin/indices/flush/ShardFlushRequest.java

```diff
@@ -53,5 +54,8 @@ public void writeTo(StreamOutput out) throws IOException {
         request.writeTo(out);
     }
 
-
+    @Override
+    public String toString() {
+        return "flush {" + shardId() + "}";
```

Comment:
I think this should also concatenate the underlying `FlushRequest` `request#toString`.